src/pretix/base/models/orders.py

```diff
@@ -2326,6 +2332,12 @@ class CancellationRequest(models.Model):
     refund_as_giftcard = models.BooleanField(default=False)
 
 
+class TicketSecretBlacklist(models.Model):
```

Comment:
You probably want to call these Denylists? :)

Reply:
Argh, thanks, yes. I planned "revocation list" which is even preciser than blacklist, its already in the commit in other places, but I forgot the model. Thanks!